src/components/calendar.rs

```diff
@@ -52,6 +65,154 @@ pub const CALENDAR_PROTOCOL_METHODS: [&str; 21] = [
     "yearOfWeek",
 ];
 
+#[derive(Debug, Clone)]
+pub struct TemporalCalendar(Ref<'static, AnyCalendar>);
```

Comment:
thought: Should this be `Calendar` or `TemporalCalendar`

Potential bikeshed. We don't really refer to any other component as `Temporal*` outside of `TemporalRoundingMode`. My thought is that at minimum the names should be consistent. `TemporalCalendar`, `TemporalDate`, etc. or `Calendar`, `Date`, etc.

While I can see the argument for a `Temporal` prefix on the options as they may be present across libraries, I'm not sure the potential for colliding names being present on the components.